Fix test-graphics `uniform_int_distribution` template parameter

The `test-graphics` project uses `uniform_int_distribution` with an unsupported type parameter of `uint8_t`:
```cpp
std::uniform_int_distribution<uint8_t> dist(0, 64);
```

----

With Visual Studio 2022 this leads to compile errors:
> C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.32.31326\include\random(1908,5): error C2338: static_assert failed: 'invalid template argument for uniform_int_distribution: N4659 29.6.1.1 [rand.req.genl]/1e requires one of short, int, long, long long, unsigned short, unsigned int, unsigned long, or unsigned long long' (compiling source file TestGraphics.cpp) [C:\projects\nas2d-core\test-graphics\test-graphics.vcxproj]
  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.32.31326\include\random(1908,5): error C2338: static_assert failed: 'note: char, signed char, unsigned char, char8_t, int8_t, and uint8_t are not allowed' (compiling source file TestGraphics.cpp) [C:\projects\nas2d-core\test-graphics\test-graphics.vcxproj]

Example logs:
https://ci.appveyor.com/project/OPU/nas2d-core/builds/44397422/job/squptfiuofhvar8w

----

Documentation does mention there is an `int` restriction on the `IntType` parameter:
https://en.cppreference.com/w/cpp/numeric/random/uniform_int_distribution
> `template< class IntType = int > class uniform_int_distribution;`
> ...
> Template parameters
> `IntType`    -    The result type generated by the generator. The effect is undefined if this is not one of `short`, `int`, `long`, `long long`, `unsigned short`, `unsigned int`, `unsigned long`, or `unsigned long long`.

--- a/test-graphics/TestGraphics.cpp
+++ b/test-graphics/TestGraphics.cpp
@@ -46,12 +46,11 @@ NAS2D::State* TestGraphics::update()
 
 	{
 		const auto jitter = [&]() -> decltype(auto) {
-			std::uniform_int_distribution<uint8_t> dist(0, 64);
-			return dist(generator);
+			return jitter_distribution(generator);
 		};
 		for (auto i = 0u; i < 2000u; ++i)
 		{
-			const uint8_t grey = jitter() * 2u + 100u;
+			const uint8_t grey = static_cast<uint8_t>(jitter()) * 2u + 100u;
 			r.drawPoint(NAS2D::Point{10 + jitter(), 250 + jitter()}, NAS2D::Color{grey, grey, grey});
 		}
 	}

--- a/test-graphics/TestGraphics.cpp
+++ b/test-graphics/TestGraphics.cpp
@@ -12,6 +12,12 @@ namespace
 {
 	std::mt19937 generator;
 	std::uniform_int_distribution<int> jitterDistribution(0, 64);
+
+
+	auto jitter()
+	{
+		return jitterDistribution(generator);
+	}
 }
 
 
@@ -48,9 +54,6 @@ NAS2D::State* TestGraphics::update()
 	r.drawImage(mOglImage, {768, 256});
 
 	{
-		const auto jitter = [&]() -> decltype(auto) {
-			return jitterDistribution(generator);
-		};
 		for (auto i = 0u; i < 2000u; ++i)
 		{
 			const uint8_t grey = static_cast<uint8_t>(jitter()) * 2u + 100u;

--- a/test-graphics/TestGraphics.cpp
+++ b/test-graphics/TestGraphics.cpp
@@ -9,7 +9,7 @@
 
 
 static std::mt19937 generator;
-static std::uniform_int_distribution<int> jitter_distribution(0, 64);
+static std::uniform_int_distribution<int> jitterDistribution(0, 64);
 
 
 TestGraphics::TestGraphics() :
@@ -46,7 +46,7 @@ NAS2D::State* TestGraphics::update()
 
 	{
 		const auto jitter = [&]() -> decltype(auto) {
-			return jitter_distribution(generator);
+			return jitterDistribution(generator);
 		};
 		for (auto i = 0u; i < 2000u; ++i)
 		{

--- a/test-graphics/TestGraphics.cpp
+++ b/test-graphics/TestGraphics.cpp
@@ -8,8 +8,11 @@
 #include <random>
 
 
-static std::mt19937 generator;
-static std::uniform_int_distribution<int> jitterDistribution(0, 64);
+namespace
+{
+	std::mt19937 generator;
+	std::uniform_int_distribution<int> jitterDistribution(0, 64);
+}
 
 
 TestGraphics::TestGraphics() :

--- a/test-graphics/TestGraphics.cpp
+++ b/test-graphics/TestGraphics.cpp
@@ -53,12 +53,10 @@ NAS2D::State* TestGraphics::update()
 	r.drawImage(mDxImage, {256, 256});
 	r.drawImage(mOglImage, {768, 256});
 
+	for (auto i = 0u; i < 2000u; ++i)
 	{
-		for (auto i = 0u; i < 2000u; ++i)
-		{
-			const uint8_t grey = static_cast<uint8_t>(jitter()) * 2u + 100u;
-			r.drawPoint(NAS2D::Point{10 + jitter(), 250 + jitter()}, NAS2D::Color{grey, grey, grey});
-		}
+		const uint8_t grey = static_cast<uint8_t>(jitter()) * 2u + 100u;
+		r.drawPoint(NAS2D::Point{10 + jitter(), 250 + jitter()}, NAS2D::Color{grey, grey, grey});
 	}
 
 	r.drawBox({10, 50, 40, 40});

